sdk/communication/communication-common/src/identifierModelSerializer.ts

```diff
@@ -96,6 +108,33 @@ export interface SerializedMicrosoftTeamsAppIdentifier {
   cloud?: SerializedCommunicationCloudEnvironment;
 }
 
+/**
+ * @hidden
+ * A Microsoft Teams Phone user who is using a Communication Services resource
+ */
+export interface SerializedTeamsExtensionUserIdentifier {
+  /**
+   * The Id of the Microsoft Teams Extension user, i.e. the Entra ID object Id of the user.
+   */
+  userId: string;
+
+  /**
+   * The tenant Id of the Microsoft Teams Extension user.
+   */
+  tenantId: string;
+
+  /**
+   * The Communication Services resource Id.
```

Comment:
```suggestion
   * The Azure Communication Services resource Id.
```
nit: as we are refering as follow already in the acs sdks